routes/web.php

```diff
@@ -132,6 +132,7 @@
         Route::get('/email', NotificationEmail::class)->name('notifications.email');
         Route::get('/telegram', NotificationTelegram::class)->name('notifications.telegram');
         Route::get('/discord', NotificationDiscord::class)->name('notifications.discord');
+        Route::get('/slack', App\Livewire\Notifications\Slack::class)->name('notifications.slack');
```

Comment:
can we change `App\Livewire\Notifications\Slack` for simplify `NotificationsSlack` ?

Reply:
Done 👍